node-operators/operators/offboarding/offboarding-process.md

```diff
@@ -0,0 +1,299 @@
+
+### Using the Symbiotic CLI {: #opt-out-network-with-cli }
+
+The Symbiotic CLI provides a straightforward way to opt out of the network. Choose the appropriate command based on your network and signing method:
+
+### Using a Ledger Device
+
+=== "MainNet"
+
+ ```bash
+ python3 symb.py opt-out-network INSERT_NETWORK_ADDRESS --ledger --ledger-account INSERT_OPERATOR_ADDRESS
+ ```
+
+=== "TestNet (Sepolia)"
+
+ ```bash
+ python3 symb.py opt-out-network {{ networks.symbiotic.contracts.sepolia.tanssi_network }} --ledger --ledger-account INSERT_OPERATOR_ADDRESS --chain sepolia
+ ```
+
+For signing with a private key:
+
+=== "MainNet"
+
+ ```bash
+ python3 symb.py opt-out-network INSERT_NETWORK_ADDRESS --private-key INSERT_PRIVATE_KEY
+ ```
+
+=== "TestNet (Sepolia)"
+
+ ```bash
+ python3 symb.py opt-out-network {{ networks.symbiotic.contracts.sepolia.tanssi_network }} --private-key INSERT_PRIVATE_KEY --chain sepolia
+ ```
+
+!!! warning
+    Note that this method requires you to expose your private key; therefore, it is not recommended.
+
+### Using Etherscan {: #opt-out-network-with-etherscan }
+
+You can interact directly with the smart contract through Etherscan using a browser wallet like MetaMask:
+
+=== "MainNet"
+
+ [Contract address: {{ networks.symbiotic.contracts.mainnet.network_registry }}](https://etherscan.io/address/{{ networks.symbiotic.contracts.mainnet.network_registry }}#writeContract){target=\_blank}
+
+=== "TestNet (Sepolia)"
+
+ [Contract address: {{ networks.symbiotic.contracts.sepolia.network_registry }}](https://sepolia.etherscan.io/address/{{ networks.symbiotic.contracts.sepolia.network_registry }}#writeContract){target=\_blank}
+
+Make sure to select **`Contract`** and **`Write Contract`** then click on Connect to Web3, and select your preferred wallet (e.g. MetaMask):
+![Connect to Web3 step](/images/node-operators/operators/offboarding/offboarding-process/offboarding-process-1.webp)
+
+1. Expand the **`optOut`** function
+2. Insert the `TANSSI_NETWORK_ADDRESS` in the **where** field (e.g, `{{ networks.symbiotic.contracts.sepolia.tanssi_network }}` on Sepolia TestNet)
+3. Click **Write** and sign the transaction
+
+![Opt out operator](/images/node-operators/operators/offboarding/offboarding-process/offboarding-process-2.webp)
+
+!!! warning
+    After submitting your opt-out transaction, save the transaction hash. You'll need this hash later for verification in the [operation offboarding form](https://www.tanssi.network/ecosystem/operator-offboarding){target=_blank}.
+
+### Using Safe for Multisig Setups {: #opt-out-network-with-safe }
+
+For [Safe](https://app.safe.global/){target=\_blank} accounts, use the Transaction Builder with these addresses:
```

Comment:
why is transaction builder capitalized? is this a UI element? should it be in bold?